recipes/pystan/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pystan" %}
+{% set version = "2.14.0.0" %}
+{% set sha256sum = "3bed255ca7d35bbd4ed0d5f30470744d9a71b06c1fa8b88281053dd430b0b1b2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+  patches:
+    - patches/0001-Disable-boost-quadmath.patch
+
+build:
+  number: 0
+  # For now, due to Travis time limit, we restrict the build matrix.
+  skip: True  # [osx and (py36 or np!=111)]
+  features:
+    - vc9   # [win and py27]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
```

Comment:
It is nice to have this for users to understand that py36 is vc14 but you can put both py35 and py36 vc14 in a single line. (No need to do this now, you can modify it later in the feedstock.)